homeassistant/components/samsungtv/quality_scale.yaml

```diff
@@ -0,0 +1,102 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: no custom actions
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow:
+    status: todo
+    comment: cleanup "pin" vin CONF_PIN
```

Comment:
We should not ask the user for a name, rather set the name to something the user can identify, but only helpers are allowed to ask for a name

Reply:
Fixed in https://github.com/home-assistant/core/pull/144225

Reply:
```suggestion
  config-flow: done
```